test/onnx/test_fx_to_onnx_with_custom_op_overload.py

```diff
@@ -0,0 +1,56 @@
+# Owner(s): ["module: onnx"]
+import onnxscript
+import pytorch_test_common
+import torch
+import torch.onnx._internal.fx.custom_operator
+from torch.onnx._internal import fx
+from torch.testing._internal import common_utils
+
+
+class TestFxToOnnx(pytorch_test_common.ExportTestCase):
+    def setUp(self):
+        super().setUp()
+        self.opset_version = torch.onnx._constants.ONNX_DEFAULT_OPSET
+
+    def test_simple_function(self):
+        # Steps to put custom op into onnx model
+        #  1. Define a custom function and register it as torch._ops.OpOverload so that
+        #     this function can pass through PyTorch 2.0's FX-generation pipeline
+        #     (e.g., dynamo.export). Otherwise, the custom function will be traced
+        #     into (or decomposed) and exporter won't see any custom functions.
+        #  2. Write a custom exporter using ONNX Script for this custom function.
```

Comment:
Can (or is) the step 2&3 be optional? i.e. If user did not perform step 2&3, will the custom function be exported as a onnx custom op?

Reply:
Interesting question. But I guess the case wouldn't make sense given that user uses custom op when there is no valid ONNX op they need?

Reply:
I don't quite follow. Using an example to illustrate my previous comment. Let's say we are wrapping the attention module with this mechanism.

without step 2&3, export as single custom op `onnx_custom::attention`.
with step 2&3, export as local function `onnx_custom::attention` with function subgraph defined by step 2.

On a side note, it is interesting to explore how to capture ^ function subgraph even without step 2. After all, the pytorch eager implementation is available. I think that should be the next step.